packages/replay-next/components/Popup.tsx

```diff
@@ -12,22 +12,19 @@ type Dismiss = () => void;
 
 export default function Popup({
   children,
+  className = "",
   clientX = null,
   containerRef = null,
-  dataTestId,
-  dataTestName = "Popup",
   dismiss,
   dismissOnMouseLeave = false,
   horizontalAlignment = "center",
   showTail = false,
-  style = "default",
   target,
-}: {
+  ...rest
+}: Omit<HTMLAttributes<HTMLDivElement>, "onClick"> & {
```

Comment:
The changes to this component aren't necessary and I _could_ remove them if anyone objects. Just seemed nice to remove some unused things and simplify this component's interface.